src/wmnds/components/disruption-indicator/large/_example.njk

```diff
@@ -0,0 +1,13 @@
+{% raw %}
+{% from "wmnds/components/disruption-indicator/large/large.njk" import wmndsDisruptionIndicatorLarge %}
+
+{{
+  wmndsDisruptionIndicatorLarge({
+    primaryText: "Good service",
```

Comment:
Can this be changed to something that is being used in an example as I don't see the words "Good service" in any example so could be confusing for someone using.

Reply:
It is used in the train & tram examples but I've changed it to "Minor disruption" anyway since that appears on all of the examples.